Examples/Scripts/Python/material_mapping.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+import os
+
+from acts.examples import (
+    Sequencer,
+    WhiteBoard,
+    AlgorithmContext,
+    ProcessCode,
+    RootMaterialTrackReader,
+    MaterialMapping,
+    JsonMaterialWriter,
+    JsonFormat,
+)
+
+import acts
+from acts import (
+    Vector4,
+    UnitConstants as u,
+    SurfaceMaterialMapper,
+    VolumeMaterialMapper,
+    Navigator,
+    Propagator,
+    StraightLineStepper,
+    MaterialMapJsonConverter,
+)
+
+from common import getOpenDataDetector
+
+
+def runMaterialMapping(
+    trackingGeometry,
+    decorators,
+    outputDir,
+    inputDir,
+    mapSurface=True,
+    mapVolume=True,
+    s=None,
+):
+    s = s or Sequencer(numThreads=1)
+
+    for decorator in decorators:
+        s.addContextDecorator(decorator)
+
+    wb = WhiteBoard(acts.logging.INFO)
+
+    context = AlgorithmContext(0, 0, wb)
+
+    for decorator in decorators:
+        assert decorator.decorate(context) == ProcessCode.SUCCESS
+
+    # Read material step information from a ROOT TTRee
+    s.addReader(
+        RootMaterialTrackReader(
+            level=acts.logging.INFO,
+            collection="material-tracks",
+            fileList=[os.path.join(inputDir, "geant-material-tracks.root")],
```

Comment:
To work with the old material writer this need to be changed to : 

```suggestion
            collection="material_tracks",
            fileList=[os.path.join(inputDir, "geant4_material_tracks.root")],
```
But I agree your syntax is better and we should change this once we had G4 to the python bindings